backend/src/main/scala/cromwell/backend/standard/StandardLifecycleActorFactory.scala

```diff
@@ -145,8 +146,9 @@ trait StandardLifecycleActorFactory extends BackendLifecycleActorFactory {
                                 serviceRegistryActor: ActorRef,
                                 ioActor: ActorRef,
                                 cacheCopyAttempt: Int,
+                                cacheHit: CallCachingEntryId,
```

Comment:
I'm sort of amazed this wasn't already available to the method. How was it copying the results across if it didn't know which hit to copy?

Assuming somewhere there must be a "this is the hit details to copy" object passed to the cache copy actor, could we smuggle this ID into that object rather than the actor `props`?

Reply:
The EJEA has already fetched the cached outputs from the database so it only needs to `makeBackendCopyCacheHit` which is just copying files, nothing to do with cache IDs. 

I could move the ID to `CopyOutputsCommand` if you have a strong preference for that, though it seems somewhat arbitrary and would then be stored in an `Option` field in the copying actor's FSM state data.